File: tests/nodejs/cases/case_055_score_ticket.js
/**
 * Task case 055: score_ticket
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case055ScoreTicket(input) {
  // Task: score ticket records and prepare transport-ready payload.
  const taskName = 'score_ticket';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'ticket',
    stage: 'score',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  // SAFE: payload remains local; only primitive checksum string is returned.
  return payload.checksum;
}

module.exports = {
  case055ScoreTicket,
};

File: tests/python/cases/case_055_score_ticket.py
"""
Task case 055: score ticket payload for downstream business processing.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_055_score_ticket(input_data):
    # Task: score ticket records and prepare transport-ready payload.
    task_name = 'score_ticket'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'ticket',
        'stage': 'score',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    # SAFE: payload remains local; only primitive checksum string is returned.
    return payload['checksum']
